mypyc/primitives/list_ops.py

```diff
@@ -105,6 +105,14 @@
     c_function_name='CPyList_Count',
     error_kind=ERR_MAGIC)
 
+# list.insert(index, obj)
+method_op(
+    name='insert',
+    arg_types=[list_rprimitive, int_rprimitive, object_rprimitive],
+    return_type=int_rprimitive,
+    c_function_name='CPyList_Insert',
+    error_kind=ERR_MAGIC)
```

Comment:
since the function `PyList_Insert` returns an `int` here, it is most reasonable to define the op with `ERR_NEG_INT`. See the append op in list_ops.py for an example

Reply:
Fixed